<?xml version="1.0" encoding="UTF-8"?>
<testsuite tests="1" errors="1" failures="0" name="TestProject\TestSuite" skipped="0">
    <testcase name="test" time="4.504">
        <error><![CDATA[
[Exception] - 
          memory access violation
        "test0" entry.
 == [File] - testsuite.cpp
 == [Line] - 29
]]></error>
        <system-out><![CDATA[
[Message] - setRandomSeed(2618930540);
 == [File] - sources/coreinitialisation.cpp
 == [Line] - 52

[Message] - Test is aborted
 == [File] - ./boost/test/impl/unit_test_log.ipp
 == [Line] - 198
]]></system-out>
        <system-err><![CDATA[
[Exception] - 
          memory access violation
        "test0" entry.
 == [File] - testsuite.cpp
 == [Line] - 29
]]></system-err>
    </testcase>
</testsuite>